resources/assets/js/app.js

```diff
@@ -13810,8 +14064,72 @@ var APPModalMTDetails = function (_APPModal) {
 
 				// title
 				$panel.append('<div class="panel-heading">' + menu.label + '</div>');
-                var $plot = $(modelHandler._img).appendTo($panel).wrapAll('<div class="panel-plot"></div>');
-				//var $plot = $('<figure class="figure"><img src="' + modelHandler._img + '" /></figure>').appendTo($panel).wrap('<div class="panel-plot"></div>');
+
+				var $plot = $(modelHandler._img).appendTo($panel).wrapAll('<div class="panel-plot"></div>');
+			}
+
+			return $panel;
+		}
+		/**
+   * CREATE PLOT PANEL
+   * create plot tab-pane for specific menu
+   * @param {array} menu
+   * @param {object} modelHandler: object of class Model
+   */
+
+	}, {
+		key: '_createModelScriptPanel',
+		value: function _createModelScriptPanel(menu, modelHandler) {
+			var O = this;
+			_log('MODAL DETAILS / _createPlotPanel');
+
+			// tab-pane
+			var $panel = $('<div class="tab-pane h-100" role="tabpanel"></div>').attr('id', menu.id);
+
+			if (modelHandler && menu.id && modelHandler._modelScript) {
+				// get panel meta
+				var panelMeta = modelHandler._panels[menu.id];
+
+				// title
+				$panel.append('<div class="panel-heading">' + menu.label + '</div>');
+				var $script = $('<pre class="brush: js;"></pre>').appendTo($panel).wrap('<div class="panel-plot"></div>');
+
+				var lines = modelHandler._modelScript.split("\n");
+				for (var i = 0; i < lines.length; i++) {
+					$('<span class="line">' + lines[i] + '</span>').appendTo($script);
+				}
+			}
+
+			return $panel;
+		}
+		/**
+   * CREATE VISUALIZATION SCRIPT
+   * create plot tab-pane for specific menu
+   * @param {array} menu
+   * @param {object} modelHandler: object of class Model
+   */
+
+	}, {
+		key: '_createVisualizationScriptPanel',
+		value: function _createVisualizationScriptPanel(menu, modelHandler) {
+			var O = this;
+			_log('MODAL DETAILS / _createPlotPanel');
+
+			// tab-pane
+			var $panel = $('<div class="tab-pane h-100" role="tabpanel"></div>').attr('id', menu.id);
+
+			if (modelHandler && menu.id && modelHandler._visScript) {
+				// get panel meta
+				var panelMeta = modelHandler._panels[menu.id];
+
+				// title
+				$panel.append('<div class="panel-heading">' + menu.label + '</div>');
+				var $script = $('<pre class="brush: js;"></pre>').appendTo($panel).wrap('<div class="panel-plot"></div>');
+
+				var lines = modelHandler._visScript.split("\n");
```

Comment:
Are we not using the CodeMirror library?

Reply:
not right now